sdk/search/search-documents/tests.yml

```diff
@@ -1,8 +1,10 @@
 trigger: none
 
-extends:
-  template: ../../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
-  parameters:
-    PackageName: "@azure/search-documents"
-    ResourceServiceDirectory: search
-    TestMinMax: true
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/search-documents"
+      ServiceDirectory: search
+      MatrixFilters:
+        - TestType=node|browser|sample
+        - DependencyVersion=^$|min|max
```

Comment:
Isn't min/max already included in the default matrix by default? I would have expected that you might need to opt-out for any cases where TestMinMax is false but not need to opt-in where it is set to true.

Reply:
Min/max is excluded by default based on the default parameter in the pipeline. It is set as opt-in in master currently.